tests/test_views_bucket.py

```diff
@@ -8,44 +8,50 @@
 
 """Test bucket related views."""
 
+import pytest
 from flask import url_for
 from testutils import login_user
 
 from invenio_files_rest.models import ObjectVersion
 
 
-def test_head(client, headers, bucket, permissions):
-    """Test checking existence of bucket."""
-    cases = [
+@pytest.mark.parametrize(
```

Comment:
replaced the for loop over `cases` in the function with parametrization; this is done a few more times in this file